src/state/swaps/swapsStore.ts

```diff
@@ -43,14 +43,14 @@ export const swapsStore = createRainbowStore<SwapsState>(
 
     flashbots: false,
     setFlashbots: (flashbots: boolean) => set({ flashbots }),
-    slippage: getDefaultSlippage(ChainId.mainnet, DEFAULT_CONFIG),
+    slippage: getDefaultSlippage(ChainId.mainnet, getRemoteConfig()),
     setSlippage: (slippage: string) => set({ slippage }),
     source: 'auto',
     setSource: (source: Source | 'auto') => set({ source }),
   }),
   {
     storageKey: 'swapsStore',
-    version: 1,
+    version: 2,
```

Comment:
not sure if we need a version bump when making changes to the store?

Reply:
only if you want to override the persisted config, in this case if you keep `version: 1` users that have already booted the app will keep the old default slippage

when do you want to get from remote config? rn it's gonna grab when the store is first created in the device and then persisted and never grab from `getRemoteConfig` again, as it's persisted

Reply:
Maybe simpler to not persist default slippage?

Reply:
Actually I don't think it should be persisted in its current form — would only make sense to me to persist if it were an explicit "default slippage" setting.